src/data/community-events.json

```
@@ -484,5 +502,14 @@
     "description": "The Ethereum dev conference in Dubai on everything DeFi, NFTs, EVM scaling, layers 2 and more with a focus on decentralization and community projects.",
     "startDate": "2023-03-14",
     "endDate": "2023-03-16"
+  },
+  {
+    "title": "ETH PRIVACY",
+    "to": "http://www.ethprivacy.net/",
```

This link redirects; we should use the final destination URL here.

```suggestion
    "to": "https://www.leadingprivacy.com/istanbul",
```